gulpfile.js

```diff
@@ -0,0 +1,49 @@
+var gulp = require('gulp');
+var uglify = require('gulp-uglify');
+var browserSync = require('browser-sync');
+var rename = require('gulp-rename');
+var minifyCss = require('gulp-minify-css');
+
+var WORDPRESSURL = "http://localhost:8888/wordpresstest/wp-admin/edit.php?post_type=snowball";
```

Comment:
This hard-coded URL doesn't work on my install, which is at http://localhost:8888/wordpress/